example/src/App.tsx

```diff
@@ -55,17 +64,17 @@ const App = () => {
 
   function attachAdListeners() {
     // Interstitial Listeners
-    AppLovinMAX.addEventListener('OnInterstitialLoadedEvent', () => {
+    AppLovinMAX.addEventListener('OnInterstitialLoadedEvent', (adInfo) => {
       setInterstitialAdLoadState(adLoadState.loaded);
 
       // Interstitial ad is ready to be shown. AppLovinMAX.isInterstitialReady(INTERSTITIAL_AD_UNIT_ID) will now return 'true'
-      var adInfo = AppLovinMAX.getAdInfo(INTERSTITIAL_AD_UNIT_ID);
-      logStatus('Interstitial ad loaded from ' + adInfo.networkName);
+      var completeAdInfo = AppLovinMAX.getAdInfo(INTERSTITIAL_AD_UNIT_ID);
+      logStatus('Interstitial ad loaded from ' + completeAdInfo.networkName);
 
       // Reset retry attempt
       setInterstitialRetryAttempt(0)
```

Comment:
This is pretty hacky and not good API design. We should align the callbacks to include `networkName`, `creativeId`, `revenue`, `placement` (basically whatever we have in `MAAd.h`.

I'll create a change shortly to reflect this.

Reply:
Can you review this PR: @alvarshahanji - Can you help me review this PR: https://github.com/AppLovin/AppLovin-MAX-React-Native/pull/35? Then I'll merge and we can update the integration here.

Reply:
@alvarshahanji  - I merged the PR ad pushed the changes to v2.0.1, if you run `npm install` where the app's `package.json` is it'll automatically install.

Reply:
@thomasmso not receiving callbacks after adInfo changes.
Changing back following locally, I could see callbacks working.
[self sendReactNativeEventWithName: name body: [self adInfoForAd: ad]];   ->
[self sendReactNativeEventWithName: name body: @{@"adUnitId" : ad.adUnitIdentifier}];

Reply:
What do the logs say?

Reply:
logs : 
 [AppLovinSdk] ERROR [ALDelegateCallbackInvoker] Unable to notify ad delegate about ad being loaded: *** -[__NSPlaceholderDictionary initWithObjects:forKeys:count:]: attempt to insert nil object from objects[3]

Reply:
Interesting, the fix didn't go in for [this line](https://github.com/AppLovin/AppLovin-MAX-React-Native/blob/master/ios/AppLovinMAX.m#L1067) - lemme push 2.0.3

Reply:
Alright can you try 2.0.3?

Reply:
Looks good with 2.0.3. Making changes in the app.